eq-author/src/components/Forms/Input/Input.test.js

```diff
@@ -80,4 +80,47 @@ describe("components/Forms/Input", () => {
       );
     });
   });
+
+  describe("Radio", () => {
+    let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(
+        <Input
+          id="my-radio-button"
+          variant="radioBox"
+          type="radio"
+          onChange={handleChange}
+          defaultChecked={false}
+        />
+      );
+    });
+
+    it("should render a radio button", () => {
+      expect(wrapper.find("input").getElement().props.type).toEqual("radio");
```

Comment:
When writing a test, try to have the assertion be readable as a sentance as well as a piece of code. It increases readability and makes it easier if someone needs to modify your test further down the line.

```suggestion
      const element = wrapper.find("input").getElement();
      expect(element.props.type).toEqual("radio");
```